### PR TITLE
Add travis-ci config

### PR DESCRIPTION
**Pull request by ivanceras**
Note: the `--locked` flag in cargo build was remove, since it was causing a build error. 
The env `secret` is also commented out

---
This change is [Reviewable](https://reviewable.io/reviews/centrifuge/substrate-pallet-multi-account/1)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +16,39 @@ branches:
   only:
   - master
 
-before_script:
-- rustup component add rustfmt
```
Comment (philipstanislaus) [inline .travis.yml:14]: Why remove this?

File: .travis.yml [hunk 2]
```diff
@@ -10,10 +16,39 @@ branches:
...
+      - libdw-dev
+      - cmake
+      - gcc
+      - binutils-dev
+      - libiberty-dev
 
-script:
-- cargo build
-- cargo test
-- cargo fmt -- --check
```
Comment (philipstanislaus) [inline .travis.yml:19]: Why remove this? Rustfmt is used intentionally here

File: .travis.yml [hunk 3]
```diff
@@ -10,10 +16,39 @@ branches:
...
+        - secure: "p1XKGf0G74t6shD456Bbc89K83lLx8M5JZpik/7gIBwnninm5FVE+8Zm9+88gjIBSlcx9Rlp2P01va6fiTGDPTk3hDmioCJIifQSrCakd6/hiANu8IVZqj9ud83lBYuWYOY2kjTtFKV4e2JO80lny2+w6+cs3vGbltTxDwF0nAJV7T1kb2RMZOzUXd1/I0mAU68hg4EZ/tLV6H5g2I+Au1k0iST3EXxYaykGXntmDR7xv6QZboFT97ZUyb2gnqqM1MRui0aVft+XaFIy2/TQah2uwZy7cM+zzhBcVUSttiQqsNN8m789fo1xgZDFUR5tPNBBiztmRnk5/DvmESyGO8yFsy4mNFVplugyNpTq+7UBBxv2yf6CU1/TEtGNvbciTKt8celZajsm+o7Z+4kxUMOCLn1sS1wcoq3cnGHItSSHM72/ph3voJpe/4UxDMcnsWfQINLm13Q0ttNmx5hqwB9KinYor76UKE3LnfdT53G3x8/pdjkdQUURNB37A6S6UWWu7zZIJ1BgxQmQPBmIhlG98zVwqFtbuqMLX29rAUGR3Zu9YGmPcmFkoL9R2vfflUrUmdG4p5S/p9+F33PFpmSzb40I0ewlvgWyOddX/jo9/WRf0N943OxWjr7OEyIwC56t/q9zpwPo14qeu75S7L8DXNPyFONHiYddO+HhzBc="
+
+before_install:
+  # Check how much space we've got on this machine.
+  - df -h
```
Comment (philipstanislaus) [inline .travis.yml:27]: We don't need this, there is nothing written to disk in the tests.

File: scripts/init.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eux
+
+RUST_TOOLCHAIN="${RUST_TOOLCHAIN:-nightly}"
+
+echo "*** Initializing WASM build environment"
```
Comment (philipstanislaus) [inline scripts/init.sh:7]: Why is this included? The module does not use WASM at all.

File: .travis.yml [hunk 5]
```diff
@@ -10,10 +16,39 @@ branches:
...
+  include:
+    - stage: "Test and build"
+      name: "Run runtime tests"
+      script: RUST_TOOLCHAIN=nightly-2020-02-17 TARGET=runtime-test ./ci/script.sh
+    - # stage name not required, will continue last one
+      name: "Build (WASM and native)"
+      script: RUST_TOOLCHAIN=nightly-2020-02-17 TARGET=build-client ./ci/script.sh
+    - stage: "Build and push Docker image"
+      if: (NOT type IN (pull_request)) AND (branch = master)
+      script: travis_wait 180 ./ci/docker.sh
```
Comment (philipstanislaus) [inline .travis.yml:39]: I don't think we should publish a docker image of a library/module, but maybe I'm missing something – what's the reasoning behind that?

**Review by philipstanislaus — CHANGES_REQUESTED** [review]
Thanks @ivanceras. 

Most of the changes you made are not applicable to this pallet though – I probably makes sense to close this PR and only include the codecov config in a new onei90-=.

**Comment by ivanceras** [comment]
I'll close this one and create a new PR.

**Pull request closed without merging**